lib/backend.js

```diff
@@ -9,16 +9,52 @@ const pickBy = require('lodash/pickBy')
 const omit = require('lodash/omit')
 const querystring = require('querystring')
 const clc = require('cli-color')
-
 require('es7-object-polyfill')
 
+process.env.AWS_SDK_LOAD_CONFIG=1
+const AWS = require('aws-sdk')
+
+/**
+ * Create the configuration for the local dynamodb instance.
+ *
+ * Region and AccessKeyId are determined as follows:
+ *   1) Look at local aws configuration in ~/.aws/credentials
+ *   2) Look at env variables env.AWS_REGION and env.AWS_ACCESS_KEY_ID
+ *   3) Use default values 'us-east-1' and 'key'
+ *
+ * @param env - the process environment
+ * @returns {{endpoint: string, sslEnabled: boolean, region: string, accessKeyId: string}}
+ */
+function loadDynamoConfig(env) {
+  const dynamoConfig = {
+    endpoint: 'http://localhost:8000',
+    sslEnabled: false,
+    region: 'us-east-1',
+    accessKeyId: 'key'
```

Comment:
So you left out the secret. I understand that this is based on reading documentation but in practice it makes it impossible to access DB in my case.

I get this error when trying to load index page:
```
{
  "error": {
    "message": "Missing credentials in config",
    "code": "CredentialsError",
    "time": "2018-10-01T07:48:43.576Z",
    "retryable": true,
    "originalError": {
      "message": "Could not load credentials from any providers",
      "code": "CredentialsError",
      "time": "2018-10-01T07:48:43.576Z",
      "retryable": true,
      "originalError": {
        "message": "Connection timed out after 1000ms",
        "code": "TimeoutError",
        "time": "2018-10-01T07:48:43.575Z",
        "retryable": true
      }
    }
  }
}
```

My case is:
 - I have `~/.aws/credentials` file but it doesn't include `default` profile credentials so it's not really used to access my local DB
 - My local DB is started with `/usr/local/bin/dynamodb_control.py --inmemory no http://localhost:8000`

It works if I add the `secretAccessKey` key to the dynamo config object. The value of the key doesn't matter as long as it's non-empty.

Reply:
Ok so in either case we need a secret key set to some constant value. I will fix this in a sec